packages/core/src/coauthor.ts

```diff
@@ -5,7 +5,7 @@ import { githubUser } from "./utils/githubUser";
 export const coauthor = (author: z.infer<typeof authorSchema>) =>
   githubUser(authorSchema.parse(author).user).then(
     (githubUser) =>
-      `Co-authored-by: ${author.name ?? githubUser.name} <${githubUser.id}+${
-        author.user
-      }@users.noreply.github.com>` as const,
+      `Co-authored-by: ${author.name ?? githubUser.name ?? author.user} <${
+        githubUser.id
+      }+${author.user}@users.noreply.github.com>` as const,
```

Comment:
This ci:test make me improve this automatically

Reply:
If there is no github display name, it appears that the implementation uses github user id instead. great !